matrixbot/plugins/wkbotsfeeder.py

```diff
@@ -93,11 +91,16 @@ def dispatch(self, handler=None):
             return  # Feeder is only updated each 'period' time
         self.lasttime = now
 
-        res = []
         for builder_name, builder in list(self.settings["builders"].items()):
             self.logger.debug("WKBotsFeederPlugin dispatch: Fetching %s ..." % builder_name)
             try:
                 build = self.get_last_build(builder)
+
+                if bool(build["complete"]) and int(build["complete_at"]) + SIX_HOURS < now:
```

Comment:
If I am right, this is  going to start spamming the channel when this condition is true Is this the desired behavior?

Reply:
Yes, you're right. In case a bot is stopped, this message will be printed out every `self.period` (1 hour). I prefer to print it only the one time.